portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,83 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/* This code script executes when the website is first loaded in  */
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+/* JavaScript "clicks" on the default tab so that content is shown when first going into the website */
 
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+
+/*global variable that gives me which slide I'm currently on in the SlideShow*/
+var slideIndex;
+
+/* Performs necessary functions when page is first loaded */
+window.onload = function( ){
+    displayDefaultSection();
+    slideIndex = 1;
+    showSlides(slideIndex);
+}
+
+/* Sets the default-tab by "clicking" on the "Background" tab, which has id="default-tab*/
+function displayDefaultSection(){
+    document.getElementById("default-tab").click();
+}
+
+/* Displays certain Sections will when a specific tab is selected at the top" */
+function displaySection(event, sectionName){
+    setToNone("section-content");
+    setToDefault("section-tab", " active");
+    /* Displays the appropriate content for the sectionName associated with the tab the user just clicked on  */
+    document.getElementById(sectionName).style.display = "block";
+
+    /* Changes the styling to active for the tab the user just clicked on */
+    event.currentTarget.className += " active";
+}
+
+/* Set all elements of given className name to style.display = none*/
+function setToNone(name){
+    var i, content;
```

Comment:
Here and others, loop iterator should be initialized in the for-loop and use let instead of var